.github/workflows/ci.yml

```diff
@@ -27,7 +27,7 @@ env:
   BUILDTOOLS_VERSION: '3.0.0'
   BUILDIFIER_SHA256SUM: 'e92a6793c7134c5431c58fbc34700664f101e5c9b1c1fcd93b97978e8b7f88db'
   BUILDOZER_SHA256SUM: '3d58a0b6972e4535718cdd6c12778170ea7382de7c75bc3728f5719437ffb84d'
-  TENSORFLOW_VERSION: 'tf-nightly'
+  TENSORFLOW_VERSION: 'tf-nightly<=2.13.0.dev20230426'
```

Comment:
Can you add a comment  
`# See https://github.com/tensorflow/tensorboard/pull/6350`
similar to [example PR](https://github.com/tensorflow/tensorboard/pull/5709)?

Please pin to a specific version instead of using "<="
see  go/tensorboard-system-hygiene#tf-nightly-is-broken

Reply:
Sure, but Isn't that what the blame is for?